### PR TITLE
Project NorthStar Azure Fixes

### PR DESCRIPTION
**Pull request by glennthomas1**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

**Review by seemantk — DISMISSED** [review]

File: dashboards/azure-container-registry/azure-container-registry.json [hunk 1]
```diff
@@ -71,8 +70,10 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT sum(azure.containerregistry.registries.SuccessfulPullCount) AS 'Total Successful Pull Count'  WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.containerregistry/registries' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```
Comment (zstix) [inline dashboards/azure-container-registry/azure-container-registry.json:74]: It looks like we have a hard-coded `accountId` here. Unless that is intentional, we should remove that.

File: dashboards/azure-container-registry/azure-container-registry.json [hunk 2]
```diff
@@ -137,8 +105,10 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT sum(azure.containerregistry.registries.TotalPullCount) AS 'Total Pull Count'  WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.containerregistry/registries' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```
Comment (zstix) [inline dashboards/azure-container-registry/azure-container-registry.json:109]: Same comment as above.

File: dashboards/azure-container-registry/azure-container-registry.json [hunk 3]
```diff
@@ -170,8 +140,10 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT sum(azure.containerregistry.registries.TotalPushCount) AS 'Total Push Count'  WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.containerregistry/registries' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```
Comment (zstix) [inline dashboards/azure-container-registry/azure-container-registry.json:144]: Same comment as above.

File: dashboards/azure-container-registry/azure-container-registry.json [hunk 4]
```diff
@@ -203,20 +175,19 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT average(azure.containerregistry.registries.StorageUsed) AS 'Avg Storage Used'  WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.containerregistry/registries' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```
Comment (zstix) [inline dashboards/azure-container-registry/azure-container-registry.json:179]: Same comment as above.

File: dashboards/azure-disk-storage/azure-disk-storage.json [hunk 5]
```diff
@@ -71,8 +70,10 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT average(azure.compute.disks.CompositeDiskReadBytes.sec) AS 'Avg Composite Disk Read Bytes/sec' WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.compute/disks' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```
Comment (zstix) [inline dashboards/azure-disk-storage/azure-disk-storage.json:74]: As with the other file, we have a hard-coded `accountId` here. Unless intentional, we should remove this.

**Review by zstix — CHANGES_REQUESTED** [review]

**Comment by glennthomas1** [comment]
Fixed all issues brought up in the comments. Please push these fixes.

**Review by seemantk — APPROVED** [review]

**Comment by glennthomas1** [comment]
All merge conflicts resolved. Should be ready to go. Please advise.

**Comment by glennthomas1** [comment]
These changes were merged in a different branch by Aarushi Jain on our team. Closing this request in favor of her changes.

**Pull request closed without merging**